src/bt_peer_connection.cpp

```diff
@@ -366,24 +408,17 @@ namespace libtorrent {
 #ifndef TORRENT_DISABLE_LOGGING
```

Comment:
the assert that ``m_sent_handshake`` is set, 2 lines above, can be removed now. it's checked in ``send_simple_message()`` already